api/src/main/java/net/kyori/adventure/title/TitleImpl.java

```diff
@@ -99,14 +80,71 @@ public int hashCode() {
     return Stream.of(
       ExaminableProperty.of("title", this.title),
       ExaminableProperty.of("subtitle", this.subtitle),
-      ExaminableProperty.of("fadeInTime", this.fadeInTime),
-      ExaminableProperty.of("stayTime", this.stayTime),
-      ExaminableProperty.of("fadeOutTime", this.fadeOutTime)
+      ExaminableProperty.of("times", this.times)
     );
   }
 
   @Override
   public String toString() {
     return this.examine(StringExaminer.simpleEscaping());
   }
+
+  static class TimesImpl implements Examinable, Times {
```

Comment:
I feel this is only going to increase boilerplate. `Title#fadeIn` becomes `Title#times#fadeIn`. Is there a pragmatic reason we need a `Times` class?